src/ai/script_ai.cpp

```diff
@@ -855,9 +921,20 @@ static int CclAiCheckForce(lua_State *l)
 }
 
 /**
+** <b>Description</b>
+**
 **  Wait for a force ready.
 **
 **  @param l  Lua state.
+**
+** Example:
+**
+** <div class="example">
+** 	<code>
+**		-- Wait the force number 1
```

Comment:
I would write "Wait until force number 1 is completed and ready for orders" to make it clear what exactly we are waiting for

Reply:
I was thinking like this:
```
-- Make the force number 1
AiForce(1,{"unit-footman",2, "unit-archer", 1})
-- Wait until force number 1 is completed and ready for orders
AiWaitForce(1)
```
Maybe is cleaner than the previous example.